HOWTO-vi.md

```diff
@@ -0,0 +1,16 @@
+Đọc tài liệu này bằng các ngôn ngữ khác: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Bahasa Indonesia](HOWTO-id.md).
+
+Chào mừng tới Free-Programming-Books! Chúng tôi chào đón những người đóng góp mới, kể cả khi những người đóng góp lần đầu thực hiện trên Github. Nếu bạn là một trong số họ, đây là một số nguồn có thể giúp:
+
+* [Giới thiệu về Pull request](https://help.github.com/articles/about-pull-requests/)
+* [Tạo một Pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+* [Github Hello World](https://guides.github.com/activities/hello-world/)
+* [Youtube - Giới thiệu Github cho người mới bắt đầu](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [Youtube - Làm thế nào để Fork một kho lưu trữ GitHub và gửi một Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [Youtube - Khóa học về Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo)
```

Comment:
Maybe better add a note that are in english. Please translate: `in english`

Example:
```markdown
* [Giới thiệu về Pull request](https://help.github.com/articles/about-pull-requests/) *(in english)*
```